funnel/templates/errors/403.html.jinja2

```diff
@@ -4,10 +4,10 @@
 {% block top_title %}
   <div class="mui--text-center">
     <img src="{{ url_for('static', filename='img/error-403.svg') }}" class="emoticon" alt="{% trans %}Unavailable{% endtrans %}"/>
-    <h1 class="mui--text-light mui--text-display1">{% trans %}Unavailable{% endtrans %}</h1>
+    <h1 class="mui--text-light mui--text-display1">{% trans %}Forbidden{% endtrans %}</h1>
```

Comment:
We used `Unavailable` because the `Forbidden` word suggests the user attempted something bad. They just opened a link. Revert this.